build.xml

```diff
@@ -123,18 +123,36 @@
 
         <exec command="composer install" dir="${drush.root}/.." passthru="true" />
 
+        <!-- Swap out the composer file for our git repository -->
+        <delete dir="${drush.root}/sites/all/modules/contrib/search_api_federated_solr" />
+        <symlink link="${drush.root}/sites/all/modules/contrib/search_api_federated_solr" target="../../../../../../../src/search_api_federated_solr-7.x" />
+
         <drush command="site-install" assume="yes">
             <option name="site-name">Drupal 7 - Federated Search</option>
             <option name="account-pass">admin</option>
             <param>standard</param>
         </drush>
+
+        <!-- The public files directory, and everything in it, needs to be world writable. -->
+        <exec command="chmod -R 777 ${drush.root}/sites/default/files" checkreturn="true" />
+
+        <!-- Copy demo files
```

Comment:
Any particular reason this is commented out? It seems like you could update the module name there and then uncomment it?